packages/core/src/components/Combobox/ComboboxHelpers/ComboboxHelpers.tsx

```diff
@@ -201,7 +201,6 @@ export function comboboxItemRenderer({
   item: IComboboxItem;
   style: CSSProperties;
   optionEvents: IComboboxOptionEvents;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
It's better to have warnings for TS (before moving to strict mode) instead of disabling the warnings